JS_lesson6/Task3.js

```diff
@@ -0,0 +1,47 @@
+// 5*. Напишите рекурсивную функцию, которая принимает на вход число и складывает его цифры. 
+// Если сумма получилась больше 9 - снова сложите цифры. И так пока, сумма не станет меньше либо равной 9. 
+//  После окончания сложений возвращает полученное число. Например при подаче числа 19 (1+9=10>9, потому 1+0=1) выводится 1
+function sumOfDigits (num) {
+    let arrayDigitsString = num.toString().split('');
```

Comment:
Convert to string without splitting if you use `for...of` loop, because both arrays and strings are iterables in JS